hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/processor/maxwell/MaxwellJsonKafkaSourcePostProcessor.java

```diff
@@ -91,7 +91,7 @@ public MaxwellJsonKafkaSourcePostProcessor(TypedProperties props) {
 
     public static final ConfigProperty<String> PRECOMBINE_FIELD_TYPE_PROP = ConfigProperty
         .key("hoodie.deltastreamer.source.json.kafka.post.processor.maxwell.precombine.field.type")
-        .defaultValue("DATA_STRING")
+        .defaultValue("DATE_STRING")
```

Comment:
Use `PreCombineFieldType.DATE_STRING.toString()` to avoid typo?

Reply:
> Use `PreCombineFieldType.DATE_STRING.toString()` to avoid typo?

done